yin_yang/plugins/gtk.py

```diff
@@ -79,7 +80,23 @@ def set_theme(self, theme: str):
             'setGtkTheme'
         )
         message.setArguments([theme])
-        connection.call(message)
+        response = connection.call(message)
+        if response.type() == QDBusMessage.MessageType.ErrorMessage:
+            logger.warning('kde-gtk-config not available, try xsettingsd')
+            xsettingsd_conf_path = Path.home() / '.config' / 'xsettingsd' / 'xsettingsd.conf'
+            if not xsettingsd_conf_path.exists():
+                logger.warning('xsettingsd not available')
+            with open(xsettingsd_conf_path, 'r') as f:
+                lines = f.readlines()
+                for i, line in enumerate(lines):
+                    if line.startswith('Net/ThemeName'):
+                        lines[i] = f'Net/ThemeName "{theme}"\n'
+                        break
+            with open(xsettingsd_conf_path, 'w') as f:
+                f.writelines(lines)
+            subprocess.run(['killall', '-HUP', 'xsettingsd'])
```

Comment:
I guess the intent here is to restart xsettingsd. Is there a nicer way of doing this, e.g. with `systemctl`?

Reply:
Its [document](https://codeberg.org/derat/xsettingsd) say I can sedding `HUG` signal to reload config but I think `systemctl restart` also works. Which one do you think is better？